components/layout_2020/dom_traversal.rs

```diff
@@ -413,8 +414,15 @@ where
                             .to_threadsafe()
                             .as_element()
                             .expect("Expected an element");
-                        let attr_val = element
-                            .get_attr(&attr.namespace_url, &LocalName::from(&*attr.attribute));
+
+                        // <https://html.spec.whatwg.org/multipage/#case-sensitivity-of-the-css-%27attr%28%29%27-function>
```

Comment:
```suggestion
```suggestion
                        // From
                        // <https://html.spec.whatwg.org/multipage/semantics-other.html#case-sensitivity-of-the-css-'attr()'-function>:

                        // > CSS Values and Units leaves the case-sensitivity of attribute names for
                        // > the purpose of the `attr()` function to be defined by the host language.
                        // > [[CSSVALUES]].
                        // > When comparing the attribute name part of a CSS `attr()`function to the
                        // names of namespace-less > attributes on HTML elements in HTML documents,
                        // the name part of the CSS `attr()` function must first > be converted to
                        // ASCII lowercase. The same function when compared to other attributes must
                        // be compared according to its original case. In both cases, to match the
                        // > values must be identical to each other (and therefore the comparison is
                        // > case sensitive).

```